eqt/ui/UIFormWidget.py

```diff
@@ -23,6 +21,13 @@ class UIFormWidget:
     |                                                          |
     +----------------------------------------------------------+
     '''
+    def __init__(self, *args, **kwargs):
```

Comment:
@casperdcl why do we add the init here with the default states? We add the other dictionary in the class below.